composer.json

```diff
@@ -20,7 +20,7 @@
       "dealerdirect/phpcodesniffer-composer-installer": true
     },
     "platform": {
-      "php": "5.6"
+      "php": "7.1"
```

Comment:
Why would this not be the same value is `require`?

Reply:
To satisfy the constraints of our current PHPUnit requirement (PHPUnit 7.5.20 asks for PHP 7.1+). In CI this isn't an issue, because if we are in a part of the test matrix specifying PHP 7.0 then we detect that and install PHPUnit 5.7 instead.

Reply:
[(Reference)](https://github.com/woocommerce/action-scheduler/blob/3.8.1/.github/workflows/pr-unit-tests.yml#L57-L80)

Reply:
You can possibly get a better sense of what's happening if you checkout the current trunk branch, remove the Composer lock file, and run `composer i`.

Reply:
Ah, I see!

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires woocommerce/woocommerce-sniffs 0.1.0 -> satisfiable by woocommerce/woocommerce-sniffs[0.1.0].
    - woocommerce/woocommerce-sniffs 0.1.0 requires php >=7.0 -> your php version (5.6; overridden via config.platform, actual: 8.3.9) does not satisfy that requirement.
  Problem 2
    - phpunit/phpunit[7.5.0, ..., 7.5.20] require php ^7.1 -> your php version (5.6; overridden via config.platform, actual: 8.3.9) does not satisfy that requirement.
    - Root composer.json requires phpunit/phpunit ^7.5 -> satisfiable by phpunit/phpunit[7.5.0, ..., 7.5.20].
```